setup.py

```diff
@@ -39,6 +39,11 @@
 metadata_path = './etc/config/metadata/runtime-images/*.json'
 settings_path = './etc/config/settings/*.json'
 
+extra_requires = {
+    'kfp-tekton': ['kfp-tekton~=0.8.1',]
+}
+extra_requires['all'] = list(set(sum(extra_requires.values(), [])))
```

Comment:
Sorry, I had a typo in my "approval" comment and find `extra_requires` confusing given the _proper_ name is `extras_require` and used as well.  As a result, I think it might be better to name this something like `runtime_extras` to convey these would only apply to runtime packages.

I do like the way it accumulates the entries and adds `'all'` though.